test/Microsoft.Azure.Devices.Edge.Test.Common/config/EdgeConfigBuilder.cs

```diff
@@ -72,7 +76,7 @@ public EdgeConfiguration Build()
             var moduleNames = new List<string>();
             var moduleImages = new List<string>();
 
-            foreach (ModuleConfiguration module in modules)
+            foreach (ModuleConfiguration module in modules.OrderBy(m => m.Name != "$edgeHub"))
```

Comment:
should it be better to handle edgeHub first and use where clause for filtering to handle other modules

Reply:
I made this change